website/docs/components/app-footer/partials/code/component-api.md

```diff
@@ -6,22 +6,22 @@ The base `AppFooter` component includes a copyright notice. It also wraps and co
 
 <Doc::ComponentApi as |C|>
   <C.Property @name="<[AF].ExtraBefore>" @type="yielded component">
-    Container that yields its content before the `<ul>` list of links and items.
+    A generic container yielded as contextual component. Its content is injected before the `<ul>` list of links and items.
   </C.Property>
   <C.Property @name="<[AF].StatusLink>" @type="yielded component">
-    The `AppFooter::StatusLink` component (see below).
+    The yielded `AppFooter::StatusLink` contextual component ([see below](#appfooterstatuslink)).
```

Comment:
I don't believe these links go anywhere. should we make them text only as in other components?

Reply:
Ahh it's a leftover of previous test. I'm going to remove it.